checks/batch/__init__.py

```diff
@@ -1,8 +1,8 @@
 # Copyright: 2019, NLnet Labs and the Internet.nl contributors
 # SPDX-License-Identifier: Apache-2.0
 BATCH_API_MAJOR_VERSION = '2'
-BATCH_API_MINOR_VERSION = '0'
-BATCH_API_PATCH_VERSION = '1'
+BATCH_API_MINOR_VERSION = '1'
+BATCH_API_PATCH_VERSION = '0'
```

Comment:
[Comment for BATCH_INDEXES below but GitHub does not allow comments on sections of the file outside of the diff area]
Indexes need to be specified for the rpki web and mail tables.